infra/dev/nginx/nginx.conf

```diff
@@ -0,0 +1,37 @@
+http {
+    server {
+        # catch all requests on port 80
+        listen 80 default_server;
+
+        # serve index.php to first requests
+        index index.php;
+
+        # serve files from public directory
+        root /var/www/public;
+
+        location / {
+            try_files $uri $uri/ /index.php?$args;
+        }
+
+        location ~ [^/]\.php(/|$) {
+            fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+            # if (!-f $document_root$fastcgi_script_name) {
+            #     return 404;
+            # }
+
+            include fastcgi_params;
+            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+            fastcgi_param PATH_INFO $fastcgi_path_info;
+            fastcgi_param PATH_TRANSLATED $document_root$fastcgi_path_info;
+
+            fastcgi_pass dev_laravel:9000;
+            fastcgi_index index.php;
+        }
+    }
+}
+
+# need an event section, even if empty
+# otherwise error on docker up
+events {
```

Comment:
Following up the other comment on this file, I didn't need this when I only had the `server { ...`